### ISSUE TITLE
FFprobe.getMediaInformation(realPath) behaves differently from FFmpeg.getMediaInformation(realPath)

### ISSUE THREAD

**Issue by RowlandOti** [tanersener/mobile-ffmpeg#344]
**Description**
I just migrated to 4.3.1 from 4.3 and was forced to change to the new method moved to FFprobe

**Expected behaviour**
Return MediaInformation without any crashes or errors, as it was working in the previous version -4.3.

**Current behaviour**
It seems to behave differently, in that now I encounter the following error:

![Desktop screenshot (1)](https://user-images.githubusercontent.com/8356008/74514914-92479780-4f1e-11ea-99a7-1f9320c3699a.png)

I run:

```java
  val info = FFprobe.getMediaInformation(realPath)
```

but the last output that it returns is the error:

```java
D/mobile-ffmpeg: Unknown output format with name 'default'
```

**Environment**
 - Platform: [e.g. Android/IOS]
 - Architecture: [arm-v7a, arm-v7a-neon, arm64-v8a, x86, x86_64, armv7, armv7s, arm64, i386, x86_64]

 - Version 4.3 and 4.3.1
 - Android Studio version  -3.5

**Other**
This same method/command was working in FFmpeg, why would it fail on migration? I am running the above command in a loop of about 500 small videos

**Comment by tanersener** [comment]
There was a bug in `4.3.1` releases about this, #324. I fixed that and re-published all `4.3.1` release packages. 

Can you clean your `gradle` cache and try again? If it doesn't solve your problem, can you provide full `logcat` logs for your app?

**Comment by RowlandOti** [comment]
This resolved it, thanks.
